### ISSUE TITLE
No error message when opening app with wrong configuration

### ISSUE THREAD

**Issue by zuccha** [R3tr0BoiDX/smwc-browser#1]
If the app is launched normally (not from the command line) and the configuration is invalid (wrong name, wrong SFC path, invalid SFC checksum), the program never starts and it doesn't provide an error message.

I had to launch the program from the command line to see the error message. People that are not familiar with the command line might not try this option and might just assume the program is broken.

I would open a window with the error message on startup if the configuration is not valid.

**Comment by R3tr0BoiDX** [comment]
Yeah, I thought about that as well, but didn't really follow that thought more deeply. Reason why I stopped didn't was, there ain't such a thing like native error dialogs I could use, and I really don't wanna get into tkinter, heh. But thinking about it now, I can just build a simple dialog with pygame, as that's already there.
